cudnn.spec

```diff
@@ -0,0 +1,26 @@
+### RPM external cudnn 8.1.1.33
+## INITENV +PATH LD_LIBRARY_PATH %i/lib64
+
+%define cudaver_maj 11.2
```

Comment:
Probably just nitpicking, but this is not the "major" CUDA version.
Can you just use `cudaver` ?

Also, @smuzaffar , is there a way to get this directly from the CUDA spec file ?

Reply:
not really, This is parsed and used by cmsBuild (even before installing dependencies), so at that time cmsBuild do not know the actual value. We can use a common file ( just like https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_11_3_X/master/cuda-flags.file ) where one can define this version and then include it in both cuda and cudnn. But this looks very much over killed for this purpose. 

  I would suggest that in`%prep` section just check if `$CUDA_VERSION` and `%{cudaver_maj}` are same (some sed/cut/grep is needed)

Reply:
OK, if it cannot be extracted form the CUDA spec file, better leave it hard coded here then - this way we don't need to rebuild cuDNN for minor updates to CUDA (i.e. 11.2.0 -> 11.2.1 --> 11.2.2).

I assume we'll notice soon enough if we do a update CUDA and fail to update CUDNN.

Reply:
OK I renamed `cudaver_maj` to `cudaver` added a check in the `%prep` section now.